.travis.yml

```diff
@@ -8,6 +8,9 @@ matrix:
     after_success:
     - pip install codecov -e .
     - coverage run --source=src/sysl -m py.test
+    - coverage run -a --source=src/sysl $(which reljam) --root $(pwd) model /test/java/relationalmodel UserModel
+    - coverage run -a --source=src/sysl $(which reljam) --root $(pwd) facade /test/java/relationalmodel UserFacade
+    - coverage run -a --source=src/sysl $(which reljam) --root $(pwd) model /test/java/tuplecomplex UserFormComplex
```

Comment:
Hmm, you are not really running tests here, are you?

Reply:
Ah. yea. I know. That was just an experiment to see if coverage increases if I run the tool directly. I will remove this.

Reply:
oh :( I got really excited about the increased coverage ;)